samza-test/src/main/python/deployment.py

```diff
@@ -76,7 +76,7 @@ def setup_suite():
 
   # Setup Samza job deployer.
   samza_job_deployer = SamzaJobYarnDeployer({
-    'config_factory': c('samza_config_factory'),
+    'config_factory': c('samza_config_loader_factory'),
```

Comment:
Does the key here need to be `config_loader_factory`? You changed `samza_job_yarn_deployer.py` to use `config_loader_factory`.

Reply:
Nice catch